02-run-tests/06-cmr-netcdf.ipynb

```diff
@@ -61,11 +61,11 @@
       "application/vnd.holoviews_exec.v0+json": "",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #16.    def get_extent(ds):
</pre><p>Nit but maybe <code>get_webmercator_extent</code> is a better name? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/developmentseed/tile-benchmarking/pull/55/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>